src/api.js

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+
+const getAllArticles = () => {
+    return axios.get("https://news-api-portfolio-piece.onrender.com/api/articles")
+    .then((articles) => {return articles.data.articles})
+};
```

Comment:
Great job setting up your axios requests in a separate file. You could use axios.create to set up a base URL in this file that you can use in every request